health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/AppModule.js

```
@@ -261,35 +182,37 @@ const AppModule = () => {
-                fill={"#00703C"}
-                width={"3rem"}
-                height={"3rem"}
-                style={{
-                  position: "absolute",
-                  left: "-10px",
-                  top: "-14px",
-                }}
+        {modulesData
+          ?.filter((module) => module?.isDisabled === "false" && module?.project?.toLowerCase() === campaignType?.toLowerCase())
+          .map((module, index) => (
+            <Card className={`module-card ${selectedModuleCodes.includes(module?.name) ? "selected-card" : ""}`}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)